cmd/sls/sls_misc_test.go

```diff
@@ -291,6 +291,7 @@ func TestDoLoadstateWithKey(t *testing.T) {
 	sampleNw := sls_common.Network{"DUMMY", "Sample dummy network", []string{}, sls_common.NetworkTypeEthernet, 0, "2014-07-16 20:55:46 +0000 UTC", nil}
 	datastore.SetNetwork(sampleNw)
 
+	// The private key support was removed, but the key can still be passed in. Now it is just ignored.
```

Comment:
do we need this function anymore?

Reply:
It causes the code to hit the two cases where it logs that the supported encryption options are ignored. i.e. cases where the request: has the private_key form, and the json has VaultData. It validates that the load successfully sets the data.

Should I remove it?

Reply:
no thats fine. thanks